src/measure_theory/l1_space.lean

```diff
@@ -104,10 +104,22 @@ end
 lemma integrable_iff_of_ae_eq {f g : α → β} (h : ∀ₘ a, f a = g a) : integrable f ↔ integrable g :=
 iff.intro (λhf, integrable_of_ae_eq hf h) (λhg, integrable_of_ae_eq hg (all_ae_eq_symm h))
 
+lemma integrable_of_ae_le {f g : α → ℝ} (h : ∀ₘ a, ∥f a∥ ≤ ∥g a∥) (hg : integrable g) : integrable f :=
```

Comment:
This should hold for `f` and `g` taking values in (possibly different) vector spaces.